packages/config/config/devices/0x0371/zwa005.json

```diff
@@ -108,24 +106,24 @@
 			]
 		},
 		"6": {
-			"label": "Basic Set Value Settings for Group 2",
-			"description": "Values to send to Group 2",
+			"label": "Basic Set Value For Group 2",
```

Comment:
I commented on another parameter that seems to be the same, but is formulated very differently. Probably makes sense to template that one?

Reply:
They're different. One is magnet and one is motion and they're different values.